app/src/ui/about/about.tsx

```diff
@@ -221,6 +225,10 @@ export class About extends React.Component<IAboutProps, IAboutState> {
       return null
     }
 
+    if (__LINUX__) {
```

Comment:
We return `null` on line 225, can we move this condition to the conditional above?